### DISCUSSION TITLE
No action for destination key "trainer.checkpoint_callback" to check its value.

### DISCUSSION THREAD

**Discussion by xiangtaowong** [Lightning-AI/pytorch-lightning#14427] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
The error is 
```
usage: simmim_pretrain_main.py [-h] [-c CONFIG] [--print_config [={comments,skip_null,skip_default}+]] {fit,validate,test,predict,tune} ...
simmim_pretrain_main.py: error: 'Configuration check failed :: No action for destination key "trainer.checkpoint_callback" to check its value.'
```
But I  already set " checkpoint_callback: null" in the yaml
Looking forward to ur reply!

**Comment by rohitgr7** [comment]
what's your PL version? if you are using the latest one, it's `enable_checkpointing` since `checkpoint_callback` deprecated in last release.

**Reply by xiangtaowong** [reply]
Thank u so much for ur reply!
I have changed to another repo which does not use PL, and u can close this issue.
Thank u again!
